nozzle.py

```diff
@@ -516,9 +521,11 @@ def gen_sponge():
         amplitude = 1.0/current_dt/25.0
         x0 = 0.05
 
-        return (amplitude * actx.np.where(
-            nodes[0] > x0, zeros + ((nodes[0] - x0) / thickness) *
-            ((nodes[0] - x0) / thickness), zeros + 0.0))
+        return amplitude * actx.np.where(
+            actx.np.greater(nodes[-1], x0),
```

Comment:
```suggestion
            actx.np.greater(nodes[0], x0),
```
Whoops this looks like a typo.